Justfile

```diff
@@ -111,9 +111,9 @@ test-rust-feature-compilation-fail target=default-target:
     {{ if os() == "linux" { "! cargo check -p hyperlight-host --no-default-features 2> /dev/null"} else { "" } }}
 
 # Test rust gdb debugging
-test-rust-gdb-debugging target=default-target: (build-rust target)
-    {{ set-trace-env-vars }} cargo test --profile={{ if target == "debug" { "dev" } else { target } }} --example guest-debugging --features gdb
-    {{ set-trace-env-vars }} cargo test --profile={{ if target == "debug" { "dev" } else { target } }} --features gdb -- test_gdb
+test-rust-gdb-debugging target=default-target features="": (build-rust target)
```

Comment:
nit shouldnt the gdb feature be always set in this recipe rather than passed. Otherwise we have a recipe called `test-rust-gdb-debugging` that only actually does that if we pass a "gdb" feature argument?

Reply:
Good catch ;)